Python/Subreddit_Image_Scrapper/README.md

```diff
@@ -0,0 +1,44 @@
+# Subreddit Image Scrapper
+
+[![forthebadge](https://forthebadge.com/images/badges/made-with-python.svg)](https://forthebadge.com)
+
+## Description
+
+- This script helps you to download images from a particular subreddit in the local system.
+- You have a choice:
+  - To select your favorite Subreddit
+  - Select the number of images you want to download (go for more than 4)
+  - Sort method you want. (New, Hot, Top)
+
+## Setup instructions
+
+```python
+pip install requests
+```
+
+```python
+python script.py
+```
+
+## Output
+
+#### Code Input
+
+![code](https://media.discordapp.net/attachments/692277853777035314/814152229988466759/unknown.png?width=1439&height=391)
+
+#### Downloaded meme 1
+
+![image1](https://cdn.discordapp.com/attachments/692277853777035314/814152598092906496/unknown.png)
```

Comment:
Can you have the image on Imgur rather than Discord?

Reply:
Done.